### PR TITLE
Change repository references to https://github.com/kubernetes/kubernetes

### PR DESCRIPTION
**Pull request by mamikonyana**
https://github.com/GoogleCloudPlatform/kubernetes started returning 404 instead of redirecting to https://github.com/kubernetes/kubernetes

### CODE REVIEW

**Comment by mamikonyana** [comment]
fixes #18596

File: Godeps/_workspace/src/github.com/spf13/cobra/bash_completions.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ cmds := &cobra.Command{
 	Short: "kubectl controls the Kubernetes cluster manager",
 	Long: `kubectl controls the Kubernetes cluster manager.
 
-Find more information at https://github.com/GoogleCloudPlatform/kubernetes.`,
+Find more information at https://github.com/kubernetes/kubernetes.`,
```
Comment (mikedanese) [inline Godeps/_workspace/src/github.com/spf13/cobra/bash_completions.md]: Don't update godeps. You need to make this change upstream.

**Comment by mikedanese** [comment]
This mitigates but does not fix #18596. #18596 broke hundreds of links across the internet that we don't have control over.

**Comment by mamikonyana** [comment]
@mikedanese out of curiosity, is there a reason to keep links to `https://github.com/GoogleCloudPlatform/kubernetes` in this repo instead of changing everything to `https://github.com/GoogleCloudPlatform/kubernetes`?

**Comment by mikedanese** [comment]
@mamikonyana No there is no reason. We should move all links that we control over to github.com/kubernetes/kubernets. Thanks for the PR.

**Comment by mikedanese** [comment]
Please reopen when you get around to fixing this up.

**Pull request closed without merging**